src/GitHub.App/Services/PullRequestService.cs

```diff
@@ -316,11 +316,6 @@ public IObservable<Unit> SwitchToBranch(ILocalRepositoryModel repository, IPullR
                         pullRequest.Head.Sha,
                         pullRequest.Base.Ref,
                         pullRequest.Head.Ref);
-
```

Comment:
This exception is now thrown from `GetPullRequestMergeBase` (with more info about what went wrong).

Reply:
It appears we still have an issue when commenting in inline mode. This wasn't the line I commented on! 😭